### DISCUSSION TITLE
Community Engagement and Knowledge Sharing

### DISCUSSION THREAD

**Discussion by volkanbicer** [MobileNativeFoundation/discussions#207] [General]
Hello everyone,

I would like to start a discussion to gain insights into how companies effectively engage, learn, and exchange knowledge within their respective iOS and Android communities.

To kick off this discussion, I'd like to propose a few questions:

   - What are some regular events or activities that you organize to engage with your community members?
   - How often do you have those events or activities?
   - What practices worked well for engagement and knowledge sharing?
   - How do you collect feedback from your community members, and how do you use it to improve the community?

Looking forward to your insights!

**Comment by GokhanArik** [comment]
I've been experimenting with this approach for a while. At our company, we hold **Android Guild** and **iOS Guild** meetings open to all engineers, and attendance is voluntary. A significant challenge we face is managing different time zones, as it's impossible to schedule a session suitable for all mobile developers due to our globally distributed teams. To address this, we rotate our weekly meetings across three different time zones.

We use the MS Planner board for organizing topics; anyone can create a card. These can be demos (excluding product demos), tips, announcements, proposals, or problems encountered. We discuss each card and attempt to identify actionable items. If a discussion is pertinent to other time zones, we retain the card in its column to allow for input from those zones.

- If an issue requires resolution, we add an action item and assign it to a volunteer. If it falls within a team's area of responsibility, we create a ticket in their project.
- For practices, tips, or announcements that need documentation or sharing, we create an action item for that purpose and later disseminate it to the community via Slack.

After the meeting, we share the recording, the cards discussed, and notes on Slack, so everyone can access a brief summary and the recording for further information. We upload the recordings to MS Stream, which enables the creation of chapters (similar to YouTube videos), facilitating easy navigation between topics. Our Slack channels are dedicated to knowledge sharing: one for Android, one for iOS, and one for Native (anything mobile). We aim to limit generic discussions to app-specific channels and encourage broader discussions in these dedicated channels, an effort that is gradually improving.

Feedback is collected either during the guild meetings or via Polly on Slack. We have implemented several changes to our process and are constantly seeking better methods.

To centralize information, we have created a dedicated Confluence space with a blog, and we encourage engineers to contribute. Documenting on Confluence simplifies sharing the information.

From my experience, creating a culture of knowledge sharing and building a community requires significant effort, consistency, and continuous improvement (especially when the teams are distributed).

I would love to hear what others are doing!
